enterprise/backend/src/metabase_enterprise/enhancements/integrations/ldap.clj

```diff
@@ -62,8 +65,8 @@
    {:keys [sync-groups?], :as settings}                                  :- i/LDAPSettings]
   (let [user (or (attribute-synced-user user-info)
                  (user/create-new-ldap-auth-user!
-                  {:first_name       first-name
-                   :last_name        last-name
+                  {:first_name       (or first-name "Unknown")
```

Comment:
Shouldn't `Unknown` be localized? I'd probably localize it with `trs` so comparison doesn't break depending on current user locale

Reply:
That's true, I didn't localize it since the equivalent code for JWT doesn't, but we should probably do it there too